tabular/setup.py

```diff
@@ -36,11 +36,8 @@
         "lightgbm>=3.3,<4.6",  # <{N+1} upper cap, where N is the latest released minor version
     ],
     "catboost": [
-        # CatBoost wheel build is not working correctly on darwin for CatBoost 1.2, so use old version in this case.
-        # https://github.com/autogluon/autogluon/pull/3190#issuecomment-1540599280
-        # Catboost 1.2 doesn't have wheel for python 3.11
-        "catboost>=1.1,<1.2 ; sys_platform == 'darwin' and python_version < '3.11'",
-        "catboost>=1.1,<1.3; sys_platform != 'darwin'",
+        "numpy>=1.25,<2.0.0",  # TODO support numpy>=2.0.0 once issue resolved https://github.com/catboost/catboost/issues/2671
+        "catboost>=1.1,<1.3",
```

Comment:
Why remove darwin logic? Does CatBoost 1.2 work on darwin now?

Reply:
Yes the support is there for Darwin, see [platform test](https://github.com/autogluon/autogluon/actions/runs/11958841532/job/33339293984#logs)
![image](https://github.com/user-attachments/assets/6b502ae1-7c70-459a-99af-292ac2b88ff3)

Reply:
great thanks!